log_entry.go

```diff
@@ -115,8 +115,27 @@ func (c *Channel) UnmarshalJSON(b []byte) error {
 	ct, ok := raw["type"]
 	if ok {
 		c.Type = ct.(string)
+		delete(raw, "type")
```

Comment:
Sorry, I'm going to be a real pain in your rear here... For some reason it didn't dawn on me that removing this would also be a breaking change, and also make it not the raw object anymore. Would you have the spare moment to squash out this line? If not I can fix it after merging this PR.

Thank you so much for this contribution.

Reply:
Not a problem at all. I'll send update shortly